src/net.cpp

```diff
@@ -2581,30 +2581,17 @@ void CNode::AskFor(const CInv& inv)
     mapAskFor.insert(std::make_pair(nRequestTime, inv));
 }
 
-CDataStream CConnman::BeginMessage(CNode* pnode, int nVersion, int flags, const std::string& sCommand)
+void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg msg)
```

Comment:
It was done this way originally so that a message could be serialized once and sent to multiple peers. If an rvalue is required, we can't do that.

I don't think there's a use-case for that at the moment, though, so changing to && is fine by me.